How to let pypi know that Jupytext 1.7.1 is the last version with explicit support of Python 2.7 and 3.5

One of my project that uses Jupytext on the CI is now failing on Python 2.7 and 3.5, with this message:
```
ERROR: Could not find a version that satisfies the requirement markdown-it-py~=0.6.0 (from jupytext->-r requirements-dev.txt (line 7)) (from versions: none)
ERROR: No matching distribution found for markdown-it-py~=0.6.0 (from jupytext->-r requirements-dev.txt (line 7))
Error: Process completed with exit code 1.
```

Between Jupytext 1.7.1 and 1.8.0 we removed the explicit support for Python 2.7 and 3.5 (but they should still work), and we replaced the conditional dependency on markdown-it-py when Python>=3.6 with a full dependency.

Still I would like to find a plan to make the `itables` CI work again on Python 2.7 and 3.5. Maybe we could just provide enough information to pypi to let it download the appropriate version of Jupytext?

@chrisjsewell , do you know how to do that? Maybe an option would be to make the dependency on markdown-it-py~=0.6.0 conditional on `python_version >= '3.6'` again in a new release of Jupytext 1.8.1? And then in Jupytext 1.8.2 we could remove the condition and take a dependency on Python>=3.6?

--- a/docs/CHANGELOG.md
+++ b/docs/CHANGELOG.md
@@ -1,6 +1,13 @@
 Jupytext ChangeLog
 ==================
 
+1.8.2 (2021-01-04)
+------------------
+
+**Changed**
+- Jupytext 1.8.2 depends on `python>=3.6`. The last version of Jupytext explicitly tested with Python 2.7 and 3.5 was Jupytext 1.7.1, cf. [#697](https://github.com/mwouts/jupytext/issues/697).
+
+
 1.8.1 (2021-01-03)
 ------------------
 

--- a/environment.yml
+++ b/environment.yml
@@ -3,7 +3,7 @@ channels:
   - defaults
   - conda-forge
 dependencies:
-  - python
+  - python>=3.6
   - jupyter
   - jupyterlab
   - pyyaml

--- a/jupytext/version.py
+++ b/jupytext/version.py
@@ -1,3 +1,3 @@
 """Jupytext's version number"""
 
-__version__ = "1.8.1"
+__version__ = "1.8.2"

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,4 @@
-#markdown-it-py~=0.6.0
 nbformat>=4.0.0
 pyyaml
 toml
+markdown-it-py~=0.6.0

--- a/setup.py
+++ b/setup.py
@@ -58,12 +58,8 @@ setup(
     ],
     entry_points={"console_scripts": ["jupytext = jupytext.cli:jupytext"]},
     tests_require=["pytest"],
-    install_requires=[
-        "markdown-it-py~=0.6.0; python_version >= '3.6'",
-        "nbformat>=4.0.0",
-        "pyyaml",
-        "toml",
-    ],
+    install_requires=["nbformat>=4.0.0", "pyyaml", "toml", "markdown-it-py~=0.6.0"],
+    python_requires="~=3.6",
     extras_require={
         # left for back-compatibility
         "myst": [],

